Master

SpanishSurfer opened the pull request:
Zerty Fix Pull Request

SpanishSurfer committed:
Town Adjustment (needs more work)

SpanishSurfer committed:
Fatigue adjustment and Pod Price decrease

SpanishSurfer committed:
Banner, Map draw, pod prices, and town links

SpanishSurfer committed:
Mission name tweak

zerty committed:
fix #62

zerty committed:
some Cleanup

zerty committed:
Removal of Online help in action menu

zerty committed:
Prevent any other action than suicide

should fix #82

zerty committed:
Prevent garbage loops

Strill trying to fix #77

zerty committed:
Weather system changed #81

zerty committed:
removed old weather file

zerty committed:
Removing some junk code

zerty committed:
Fix uav loadout #85

zerty committed:
Removed deprecated file

zerty committed:
Removing old fatigue script

zerty committed:
Removing unused side CAS patrols

zerty committed:
Trophy code cleanup

zerty committed:
Should fix #78 , to test out

zerty committed:
Fix for #63

giving access to the air loadout menu only if in range of an ammo depot

SpanishSurfer committed:
Air Fact Upgrade Fix

SpanishSurfer committed:
Zerty Fixes

SpanishSurfer committed:
Air Upgrade Fix

SpanishSurfer committed:
Test

SpanishSurfer committed:
Merge remote-tracking branch 'SpanishSurfer/BECTI_ZERTY_OFPS.australia'

SpanishSurfer committed:
Test2

SpanishSurfer committed:
Merge remote-tracking branch 'SpanishSurfer/master'

zerty committed:
Forgot to remove a def

zerty committed:
Small hud error

zerty committed:
First changes

zerty committed:
Continued

zerty committed:
Merge branch 'master' into Gear-access-temp-branch

zerty committed:
Part 2

zerty committed:
Merge pull request #86 from zerty/Gear-access-temp-branch

Gear access temp branch

zerty committed:
Prevent Airdrop circle to show when not asking for a halo

MarkusNemesis commented:
Good to see things moving along still with this community, and to see you all using the power of Git to it's fullest. 

Keep up the good work.

zerty committed:
New notification framework

zerty committed:
Some bug

zerty committed:
Merge branch 'master' into Change-notif_framework

zerty committed:
Additionnal messages

zerty committed:
reduced number of loops on connections

zerty committed:
Merge branch 'master' into Change-notif_framework

zerty committed:
Additionnal stuff

zerty committed:
Removed debug info

zerty committed:
Spelling

zerty committed:
Durations change

SpanishSurfer committed:
Merge branch 'master' into zerty/master

SpanishSurfer committed:
Merge remote-tracking branch 'upstream/master'

SpanishSurfer committed:
Merge branch 'master' into SpanishSurfer/master

SpanishSurfer committed:
Removal of 9.3mm silencer in OPFOR

SpanishSurfer committed:
Merge branch 'master' of https://github.com/SpanishSurfer/becti_zerty_OFPS.Altis

SpanishSurfer committed:
Merge remote-tracking branch 'upstream2/master'

zerty committed:
Mobile gear fix

zerty committed:
Logic fail

zerty committed:
Fixing conditions of factory access

zerty committed:
Merge pull request #92 from zerty/Change-notif_framework

Change notif framework

zerty committed:
Upgrade notifications changed

zerty committed:
Change of Max. Notif shown

zerty committed:
Some tac hud adjustmets

- do not show the group units inside the player vehicle
- show friendly uav laser target even if the controling player is inside
a vehicle

zerty committed:
Fix for #34

zerty committed:
More debug information to check the approvement process

zerty committed:
Notif change, less intrusive

zerty committed:
Some problem with map

SpanishSurfer committed:
Net Range Increase/more upgrades, wreck change, town rework

zerty committed:
revert on hud modif

zerty committed:
Cost was hidden for vehicles

SpanishSurfer committed:
Map Fix

SpanishSurfer committed:
Merge remote-tracking branch 'zertyup/master'

SpanishSurfer committed:
Merge remote-tracking branch 'spanishup/master'

zerty committed:
Removing showing units in your own vehicle

Also vehicle you own are now marked in yellow

zerty committed:
Test Change in notif, removing timeout of messg

zerty committed:
Trying To optimize groups & join process

zerty committed:
HQ keys not complient with other vehcile keys

zerty committed:
Changing position of HUD_NOTIFICATION in init process

SpanishSurfer committed:
Merge remote-tracking branch 'zertyup/master'

SpanishSurfer committed:
Merge remote-tracking branch 'spanishup/master'

SpanishSurfer committed:
Base Startup decrease

SpanishSurfer committed:
Merge branch 'SpanishSurfer/master'
